### PR TITLE
LADX: ghost fills ammo to initial max

### PR DESCRIPTION
**Pull request by threeandthreee**
## What is this fixing or adding?
Change: Make the Ghost in your starting house fill up your consumables (Magic Powder, Bombs, Arrows) up to initial max (20/30/30) instead of just up to 10.

The reason is that in Vanilla Magic Powder is logically restored through Trendy Minigame and Bomb & Arrows are filled with the Shop. But with Entrance Shuffle and minimal accessibility, this cannot be guaranteed. Even on full accessibility it could create scenarios where refillables are only possible up to 10 but logic requires more than 10 consumables to get from start to a specific check.

TL;DR: Currently, AP can create logically impossible seeds with ER if logic requires a lot of consumables. The ghost-refill-change will fix this. Thus, this will also not be a YAML-option but a specific change.

The original maintainer, zig, also wanted to do this months ago but didn't get to it. 

## How was this tested?
Generated locally and tested with item cheating.

### CODE REVIEW

**Review by NewSoupVi — APPROVED** [review]

**Comment by kyle-basis** [comment]
🙏
